checker.go

```diff
@@ -26,6 +26,19 @@ var loggerCheckersByName = loggerCheckerMap{
 			"(k8s.io/klog/v2.Verbose).ErrorS",
 		),
 	},
+	"zap": {
+		packageImport: "go.uber.org/zap",
+		funcs: newStringSet(
+			"(*go.uber.org/zap.SugaredLogger).With",
```

Comment:
For receivers, there is not need to add `*`, which is invalid

Reply:
You can run `make test` prior submitting

Reply:
OK, I'll check this later

Reply:
@ttys3 your updated checks looks good at a glance. I'll deep into it later